great_expectations/expectations/metrics/query_metrics/query_template_values.py

```diff
@@ -28,7 +28,10 @@ class QueryTemplateValues(QueryMetricProvider):
     @classmethod
     def get_query(cls, query, template_dict, selectable):
         template_dict_reformatted = {
-            k: str(v).format(active_batch=selectable) for k, v in template_dict.items()
+            k: sa.text(v).format(active_batch=selectable)
+            if isinstance(v, int)
```

Comment:
Do we care about floats?
If so you can just check for float. Int is a subclass of float.